src/parser/PropertyPath.h

```diff
@@ -24,31 +22,31 @@ class PropertyPath {
     ZERO_OR_ONE
   };
 
-  PropertyPath() : _operation(Operation::IRI) {}
-  explicit PropertyPath(Operation op) : _operation(op) {
+  PropertyPath() : operation_(Operation::IRI) {}
+  explicit PropertyPath(Operation op) : operation_(op) {
     if (op == Operation::ZERO_OR_MORE || op == Operation::ZERO_OR_ONE) {
-      can_be_null_ = true;
+      canBeNull_ = true;
     }
   }
   PropertyPath(Operation op, std::string iri,
                std::initializer_list<PropertyPath> children);
 
   static PropertyPath fromIri(std::string iri) {
-    PropertyPath p(PropertyPath::Operation::IRI);
-    p._iri = std::move(iri);
+    PropertyPath p(Operation::IRI);
+    p.iri_ = std::move(iri);
     return p;
   }
 
-  static PropertyPath fromVariable(Variable var) {
-    PropertyPath p(PropertyPath::Operation::IRI);
-    p._iri = std::move(var.name());
+  static PropertyPath fromVariable(const Variable& var) {
+    PropertyPath p(Operation::IRI);
+    p.iri_ = var.name();
     return p;
   }
 
   static PropertyPath makeWithChildren(std::vector<PropertyPath> children,
-                                       PropertyPath::Operation op) {
-    PropertyPath p(std::move(op));
-    p._children = std::move(children);
+                                       const Operation op) {
```

Comment:
```suggestion
                                       const Operation& op) {
```

Reply:
No, trivially copyable small types can be passed by value, the pointer is an overhead:)
The const is for the usage within the function:)